src/main/scala/org/kframework/minikore/MiniKore.scala

```diff
@@ -29,7 +29,7 @@ object MiniKore {
   case class Axiom(pattern: i.Pattern, att: Attributes) extends Sentence
 
   case class Variable(_1: i.Name, _2: i.Sort) extends i.Variable {
-    def apply(_1: i.Name, _2: i.Sort): Variable = Variable(_1, _2)
+    def apply(_1: i.Name, _2: i.Sort): Variable = new Variable(_1, _2)
```

Comment:
Why does this class define an `apply` method? This method belongs to the *companion object*, not every instance of the class, and is already generated there by the case class.